frontend/packages/console-shared/src/images/skeleton-chart-cpu.svg

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should we call this file skeleton-chart.svg instead of skeleton-chart-cpu.svg? It's used for all charts.

Reply:
Agree with @spadgett on the file name.  Otherwise LGTM.

Reply:
Renamed and deleted old png